srcpkgs/NetworkManager/template

```diff
@@ -51,6 +52,10 @@ patch_args=-Np1
 build_options="gir elogind"
 build_options_default="gir elogind"
 
+pre_check() {
+	echo 'dcb30309cd6c8b7cc20383d85a5c7012' > /etc/machine-id
```

Comment:
Could you add a comment on top about this being a random or specific value?

Also, I think it would be correct to remove the file in `post_check`.

Reply:
The value is random, copied from the elogind template. I will add the comment and remove the file.